build/build/src/engine/context.rs

```diff
@@ -523,24 +525,17 @@ impl RunContext {
         }
 
 
-        // if build_native_runner {
-        //     let factorial_input = "6";
-        //     let factorial_expected_output = "720";
-        //     let output = Command::new(&self.repo_root.runner)
-        //         .args([
-        //             "--run",
-        //
-        // self.repo_root.engine.runner_native.src.test.resources.factorial_enso.as_str(),
-        //             factorial_input,
-        //         ])
-        //         .env(ENSO_DATA_DIRECTORY.name(), &self.paths.engine.dir)
-        //         .run_stdout()
-        //         .await?;
-        //     ensure!(
-        //         output.contains(factorial_expected_output),
-        //         "Native runner output does not contain expected result."
-        //     );
-        // }
+        if build_native_runner {
+            runner_sanity_test(&self.repo_root, None).await?;
```

Comment:
Please, add the following debug output:
```suggestion
            debug!("Building and testing native engine runners");
            runner_sanity_test(&self.repo_root, None).await?;
```
So that we can easily distinguish this part in the output.